src/database.ts

```diff
@@ -1348,12 +1349,29 @@ class Database extends GrpcServiceObject {
       delete gaxOpts.pageToken;
     }
 
-    return this.requestStream({
-      client: 'SpannerClient',
-      method: 'listSessionsStream',
-      reqOpts,
-      gaxOpts,
-    });
+    // eslint-disable-next-line @typescript-eslint/no-this-alias
+    const self = this;
+    const transform = function (
+      this: Transform,
+      chunk: databaseAdmin.spanner.v1.ISession,
+      enc: string,
+      callback: Function
+    ) {
+      const session = self.session(chunk.name!);
+      session.metadata = chunk;
+      this.push(session);
```

Comment:
I believe `callback(null, session)` would be our usual pattern.

Reply:
This seems neater.
Thanks!